yarn.lock

```diff
@@ -3563,7 +3563,7 @@ debug@3.1.0:
   dependencies:
     ms "2.0.0"
 
-debug@^3.0.0, debug@^3.1.1, debug@^3.2.5, debug@^3.2.6:
+debug@^3.0.0, debug@^3.1.1, debug@^3.2.5:
```

Comment:
@stacyk I'm not sure why your last few commits have included changes to `yarn.lock`, but I don't think it's intentional?